src/main/java/de/tum/in/www1/artemis/web/rest/QuizExerciseResource.java

```diff
@@ -124,7 +124,7 @@ public ResponseEntity<QuizExercise> createQuizExercise(@RequestBody QuizExercise
             return ResponseEntity.badRequest().headers(HeaderUtil.createFailureAlert(applicationName, true, ENTITY_NAME, "invalidQuiz", "The quiz exercise is invalid")).body(null);
         }
 
-        quizExercise.validateScoreSettings();
+        quizExercise.validateDates();
```

Comment:
Is validating the score settings no longer necessary, or has it been moved elsewhere?

Same in line 165.

Reply:
Not so much no longer necessary as has never been necessary as you can't actually set the total number of points directly anyways and the points cannot have an invalid value.

Reply:
It was supposed to be `validateGeneralSettings` thanks for the catch, I fixed it.